terminal_connectiontoken.go

```diff
@@ -0,0 +1,13 @@
+package stripe
+
+// TerminalConnectionTokenParams is the set of parameters that can be used when creating a terminal connection token.
+type TerminalConnectionTokenParams struct {
+	Params          `form:"*"`
+	OperatorAccount *string `form:"operator_account"`
+}
+
+// TerminalConnectionToken is the resource representing a Stripe terminal connection token.
+type TerminalConnectionToken struct {
+	Object string `json:"object"`
+	Secret string `json:"connection_token"`
```

Comment:
Can you change `json:"connection_token"` here to `json:"secret"`?

Reply:
Good catch! will do